source/ember-cli/index.md

```diff
@@ -0,0 +1,58 @@
+Installing Ember is *easy*! And our new install process includes *Ember CLI*, Ember's build tool
```

Comment:
I'm not a huge fan of new here.  In 6 months, this guide will still be live and it will not be "new"....